core/src/main/scala/org/apache/spark/FutureAction.scala

```diff
@@ -58,7 +58,7 @@ trait FutureAction[T] extends Future[T] {
    *
    * @param atMost maximum wait time, which may be negative (no waiting is done), Duration.Inf
    *               for unbounded waiting, or a finite positive duration
-   * @throws Exception exception during action execution
+   * @note Throws `Exception` exception during action execution
```

Comment:
Ugh, this still generates warnings in unidoc? Does it actually render correctly anyway?

If it renders correctly, I'm inclined to keep it, as long as it doesn't fail the build.
If it didn't render correctly to begin with, OK.